src/librustdoc/html/format.rs

```diff
@@ -207,12 +224,13 @@ impl clean::Generics {
 /// * The Generics from which to emit a where-clause.
 /// * The number of spaces to indent each line with.
 /// * Whether the where-clause needs to add a comma and newline after the last bound.
-crate fn print_where_clause<'a, 'tcx: 'a>(
+crate fn print_where_clause<'a, 'tcx: 'a, 'b: 'a>(
     gens: &'a clean::Generics,
     cache: &'a Cache,
     tcx: TyCtxt<'tcx>,
     indent: usize,
     end_newline: bool,
+    relative_to: &'b Vec<String>,
```

Comment:
What happens if you use `&'a Vec` instead?

Reply:
It works fine. a78af2d4073